src/lib/openapi/spec/project-status-schema.ts

```diff
@@ -0,0 +1,25 @@
+import type { FromSchema } from 'json-schema-to-ts';
+import { projectActivitySchema } from './project-activity-schema';
+
+export const projectStatusSchema = {
+    $id: '#/components/schemas/projectStatusSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: ['activities'],
+    description:
+        'Schema representing the overall status of a project, including an array of activity records. Each record in the activity array contains a date and a count, providing a snapshot of the project’s activity level over time.',
+    properties: {
+        activities: {
```

Comment:
I'm not sure `activities` is a good name for this property. It took me a good while into reading this PR before I realized that activities weren't things such as "flag created", "strategy added", but instead an aggregated list of such activities (presumably?) divided into per-day data.

Not suuper important (and as long as the endpoint is unstable we can change it), but maybe we could use a different wording. I mean, this isn't really a list of activities, but a list of dates with activity information. Would `activityCountByDate` be too wordy?